src/Symfony/Bridge/Doctrine/PropertyInfo/DoctrineExtractor.php

```diff
@@ -234,7 +243,22 @@ private function getPhpType($doctrineType)
                 return Type::BUILTIN_TYPE_RESOURCE;
 
             case DBALType::OBJECT:
+            case DBALType::DATE:
+            case DBALType::DATETIME:
+            case DBALType::DATETIMETZ:
+            case 'vardatetime':
+            case DBALType::TIME:
+            case 'date_immutable':
+            case 'datetime_immutable':
+            case 'datetimetz_immutable':
+            case 'time_immutable':
+            case 'dateinterval':
                 return Type::BUILTIN_TYPE_OBJECT;
+
+            case DBALType::TARRAY:
```

Comment:
⚠️ Doctrine ArrayType uses serialize / unserialize, so I think the type cannot be safely guessed. I have to test it and deprecate it if that's the case. This is why the JSON Doctrine type is not supported by this extractor I guess (it uses json_encode / json_decode).

Reply:
I just realize 'json' is the new name of 'json_array', so it has the same problem IMO.